lib/yt/actions/get.rb

```
@@ -14,7 +14,7 @@ def get
     private
 
       def get_request(params = {})
-        @list_request = Yt::Request.new(params).tap do |request|
+        @get_request = Yt::Request.new(params).tap do |request|
```

Nit: This assignment serves no purpose right?

---

Yes you are right, the `@get_request` is not used anywhere. We need to raise an error using it in the message, when request failed.